web/src/main/webapp/index.html

```diff
@@ -8,11 +8,11 @@
         <link rel="search" type="application/opensearchdescription+xml" title="GraphHopper Maps" href="opensearch.xml"/>
         <title>Driving Directions - GraphHopper Maps</title>
         <meta name="description" content="A fast route planner for biking, hiking and more! Based on OpenStreetMap including elevation data. Try out for free!"/>
-        <link rel="stylesheet" href="css/leaflet.css?v=1.0.2" />
+        <link rel="stylesheet" href="css/leaflet.css?v=1.0.3" />
         <link rel="stylesheet" href="css/leaflet.elevation-0.0.4.css" />
         <link rel="stylesheet" href="css/leaflet_numbered_markers.css" />
         <link rel="stylesheet" href="css/leaflet.contextmenu.css" />
-        <link rel="stylesheet" href="css/leaflet.loading.css" />
+        <link rel="stylesheet" href="css/leaflet.loading.css?v=0.1.24" />
         <link rel="stylesheet" href="css/ui-lightness/jquery-ui.min.css" />
         <script type="text/javascript" src="js/main.js?v=0.9.0"></script>
```

Comment:
Would we be in the need to change the v=xy to e.g. 0.9.1 so that we force a reload for everyone?